packages/files/_test.pony

```diff
@@ -138,6 +139,34 @@ class iso _TestDirectoryOpen is UnitTest
       h.assert_true(tmp.remove())
     end
 
+class iso _TestDirectoryFileOpen is UnitTest
+  fun name(): String => "files/Directory.open-file"
+  fun apply(h: TestHelper) =>
+  try
+    // make a temporary directory
+    let dir_path = FilePath.mkdtemp(
+      h.env.root as AmbientAuth,
+      "tmp.directory.open-file")?
+    try
+      let dir = Directory(dir_path)?
+
+      // create a file (rw)
+      let created:File = dir.create_file("created")?
+      h.assert_true(created.valid())
+      created.dispose()
+
+      // open a file (ro)
+      let readonly:File = dir.open_file("created")?
```

Comment:
For compliance with the standard library style guide, can you add a space here after the `:`?

Same comment for line 154 above.